rules/unicorn.js

```diff
@@ -113,10 +60,6 @@ export default {
         },
       },
     ],
-    'unicorn/require-array-join-separator': 'error',
-    'unicorn/require-number-to-fixed-digits-argument': 'error',
-    // No good use cases yet.
-    'unicorn/string-content': 'off',
-    'unicorn/throw-new-error': 'error',
+    'unicorn/switch-case-braces': ['error', 'avoid'],
```

Comment:
the default is always require braces, and from looking at our codebases it seems like it's not how we usually write it. changed to `avoid` for this reason, see [docs for details](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/switch-case-braces.md)